### DISCUSSION TITLE
[FEATURE REQUEST] simple output from running tasks

### DISCUSSION THREAD

**Discussion by johnvlittle** [dbeaver/dbeaver#35591] [Ideas]
When your run a task, e.g. to select the last version number of a version table, you can specfy the output goes into a log file.
The problem is the log file is packed with debug info, making the results hard to see and use.
What we want form the log file is just the database name and the output from running the sql (a version number in this case), assuming no errors.

this could even be a csv, so we can import the output (sort of like a mutli-database reporting tool).

Use cases for this include
- checking if a particular index is on each database.
- checking the max id of a particular table 
- seeing what flyway or liquibase version number is on each db.
NOTE: we have a DB for each brand, and in theory each should have the same schema, but sometimes they are not the same.

**Comment by pinki840** [comment]
@ksourabh321     hiiiii
